packages/wrangler/src/constellation/index.ts

```diff
@@ -0,0 +1,75 @@
+import * as CreateProject from "./createProject";
+import * as DeleteProject from "./deleteProject";
+import * as DeleteProjectModel from "./deleteProjectModel";
+import * as ListCatalog from "./listCatalog";
+import * as ListModel from "./listModel";
+import * as ListProject from "./listProject";
+import * as ListRuntime from "./listRuntime";
+import * as UploadModel from "./uploadModel";
+import type { CommonYargsArgv } from "../yargs-types";
+
+export function constellation(yargs: CommonYargsArgv) {
+	return yargs
+		.command("project", "Manage your projects", (constProjYargs) => {
+			return constProjYargs
+				.command(
+					"list",
+					"List your projects",
+					ListProject.Options,
+					ListProject.Handler
+				)
+				.command(
+					"create <name> <runtime>",
+					"Create project",
+					CreateProject.Options,
+					CreateProject.Handler
+				)
+				.command(
+					"delete <name>",
+					"Delete project",
+					DeleteProject.Options,
+					DeleteProject.Handler
+				);
+		})
+		.command("model", "Manage your models", (constModelYargs) => {
+			return constModelYargs
+				.command(
+					"upload <name> <modelName> <modelFile>",
+					"Upload a model for an existing project",
+					UploadModel.Options,
+					UploadModel.Handler
+				)
+				.command(
+					"list <name>",
+					"List models of a project",
+					ListModel.Options,
+					ListModel.Handler
+				)
+				.command(
+					"delete <name> <modelName>",
+					"Delete a model of a project",
+					DeleteProjectModel.Options,
+					DeleteProjectModel.Handler
+				);
+		})
+		.command(
+			"catalog",
+			"Check the curated model catalog",
+			(constCatalogYargs) => {
+				return constCatalogYargs.command(
```

Comment:
If there's just one list command here, could this just be `wrangler constellation catalogs`?

Reply:
It's still early days to be sure if there won't be new commands here so we prefer to leave the option open.